packages/drawer/Drawer.tsx

```diff
@@ -22,28 +24,41 @@ export class Drawer extends Component<DrawerProps> {
   static get props() {
     return {
       position: prop.string( {
-        attribute: true
+        attribute: {
+          source: true
+        }
       } ),
       visible: prop.boolean( {
-        attribute: true
+        attribute: {
+          source: true
+        }
+      } ),
+      floating: prop.boolean( {
+        attribute: {
+          source: true
+        },
+        initial: true
       } )
     };
   }
 
   position = 'top';
```

Comment:
@Hottel r eminded me off the `default` and we are already using it, so it would be better to use it to be consistent.

I missed that at first - sorry...